requirements.txt

```diff
@@ -166,14 +166,16 @@ django-taggit==1.5.1
 django-templated-mail==1.1.1
     # via djoser
 django-treebeard==4.5.1
-    # via wagtail
+    # via
+    #   -r requirements.in
+    #   wagtail
 django-user-agents==0.4.0
     # via -r requirements.in
 django-webpack-loader==1.4.1
     # via
     #   -r requirements.in
     #   mitol-django-common
-djangorestframework==3.12.4
+djangorestframework==3.14.0
```

Comment:
Was this upgrade intentional?

Reply:
It was but it's not necessary anymore, i'll revert.